angular/src/account/login/login.service.ts

```diff
@@ -33,7 +33,9 @@ export class LoginService {
 
     authenticate(finallyCallback?: () => void): void {
         finallyCallback = finallyCallback || (() => { });
-
+        
+        this.authenticateModel.rememberClient = this.rememberMe;
```

Comment:
No need this line. Token expiration is set here: https://github.com/aspnetboilerplate/module-zero-core-template/blob/master/angular/src/account/login/login.service.ts#L66

We are setting it at client-side. So no need to send it to server.

Reply:
@alirizaadiyahsi thank you!